databook/glossary.md

```diff
@@ -134,6 +138,7 @@ Neuropixels
 Ophys
   Shorthand for optical physiology, often in reference to {term}`Two-photon calcium imaging`.
 
+Optotagging
```

Comment:
I think Optotagging needs a separate definition as they aren't synonymous. 
Optotagging is using optogenetics in order to identify neurons that belong to a specific subpopulation. (and link to the optotagging section in the background)

Reply:
revised